spec/index.bs

```diff
@@ -557,8 +557,12 @@ Note: An "[=agent=]" refers to a JavaScript "thread" (i.e. main thread, or Web W
 A <dfn dfn>WebGPU interface</dfn> is an exposed interface which encapsulates an [=internal object=].
 It provides the interface through which the [=internal object=]'s state is changed.
 
-As a matter of convention, if a [=WebGPU interface=] is referred to as [=invalid=],
-it means that the [=internal object=] it encapsulates is [=invalid=].
+As a matter of convention, a [=WebGPU interface=] is considered [=invalid=]
+if and only if the [=internal object=] it encapsulates is [=invalid=].
+
+Issue: This convention can't really be used in formal contexts, because interfaces are
+content-timeline and internal objects are device-timeline.
+We can probably remove this convention eventually once other spec language is cleaned up.
```

Comment:
why do we need this convention today?

Reply:
Various places in the spec say things like "GPUBuffer is invalid" and we should ideally clean those up to say what they mean. Although we could still delete the convention text now if we want.

Reply:
It's indeed confusing when you consider content vs device timeline. So if we can remove it, let's try to do it

Reply:
It won't be easy to fully clean up the spec (I don't see how we can do it without finally fully resolving the open question of how to spec which timeline different states exist on). I'm only thinking we could delete some text here (this issue and the preceding paragraph) to make it more handwavy until we do so.